intro-HPC/ch_troubleshooting.tex

```diff
@@ -64,3 +64,35 @@ \section{Warning message when first connecting to new host}
 \else
 \firsttimeconnection
 \fi
+
+\ifusinglmod
+
+\section{Module conflicts}
+
+Modules that are loaded together must use the same toolchain version:
+it is impossible to load two versions of the same module. In the following
+example, we try to load both the \verb|intel-2018a| and \verb|intel-2017a|
```

Comment:
we try to load a module that uses the `intel-2018a` toolchain together with one that uses the `intel-2017a` toolchain: